client/src/app/management/components/meeting-edit/meeting-edit.component.html

```diff
@@ -54,6 +54,20 @@ <h2 *ngIf="!isCreateView">{{ editMeetingLabel | translate }}</h2>
         </mat-form-field>
 
         <mat-form-field>
+            <mat-label> {{ 'Participants' | translate }}</mat-label>
+            <os-search-value-selector
+                formControlName="userIds"
+                [multiple]="true"
+                placeholder="{{ 'Participants' | translate }}"
+                [inputListValues]="committee?.users"
+            ></os-search-value-selector>
+        </mat-form-field>
+
+        <!-- server bug -->
+        <!-- <mat-checkbox disabled formControlName="set_as_template">Set as template</mat-checkbox> -->
+
+        <!-- Permission: CML.can_manage || OML.can_manage_organisation -->
```

Comment:
does not fit to the permission check below.

Reply:
Why? When checked for any CML-permission, `OML.can_manage_organisation` or higher satisfy also that permission.

Reply:
Yes, you are right. Please adjust the comment to sound like `Permission: CML.can_manage (implicit: or OML.can_manage_organisation or higher)`